src/components/ManageHouses.vue

```diff
@@ -64,6 +77,7 @@ export default defineComponent({
     EditHouseForm,
     ListOfHouses,
     AddHouseForm,
```

Comment:
AddHouseForm and EditHouseForm can be removed

Reply:
I know it, thanks) the PR is just a draft, like first attempt to make one component